src/components/edit/event/EventDialog.js

```diff
@@ -131,6 +137,16 @@ class EventDialog extends Component {
             setStartDate(DEFAULT_START_DATE)
             setEndDate(DEFAULT_END_DATE)
         }
+
+        // Set org unit tree roots as default
+        if (!rows && orgUnits.roots) {
+            if (orgUnits.roots) {
```

Comment:
Is check on 143 needed since it's already checked on 142?

Reply:
My bad - fixed!